composer.json

```diff
@@ -16,5 +16,8 @@
         "psr-4": {"Tests\\": "tests/"}
     },
     "minimum-stability": "dev",
-    "require": {}
+    "require": {},
+    "require-dev": {
+        "phpunit/phpunit": "^8"
```

Comment:
I'd appreciate if we can use a stable version, say "8" instead of "^8"

Reply:
Good idea, I've merged in your changes into my branch, removing composer.json from my PR :+1: